prov/rxm/src/rxm_init.c

```diff
@@ -284,6 +284,14 @@ RXM_INI
 			"Messages of size greater than this (default: 256 Kb) "
 			"would be transmitted via rendezvous protocol.");
 
+	fi_param_define(&rxm_prov, "sar_min_seg_num", FI_PARAM_SIZE_T,
+			"Defines the minimum number of segments that are used "
+			"in SAR protocol (default: 4).");
+
+	fi_param_define(&rxm_prov, "sar_max_seg_size", FI_PARAM_SIZE_T,
+			"Defines the maximum size of segments that are used "
+			"in SAR protocol (default: = FI_OFI_RXM_BUFFER_SIZE).");
```

Comment:
Can we add to the help text that user can't set a max size > FI_OFI_RXM_BUFFER_SIZE ?